Libraries/Opc.Ua.Client/Session.cs

```diff
@@ -1312,7 +1315,7 @@
                 {
                     AddSubscription(subscription);
                 }
-
+                m_lastKeepAliveTime = DateTime.UtcNow;
```

Comment:
>m_lastKeepAliveTime [](http://example.com/codeflow?start=16&length=19)

not sure what loading subscription has to do with keep alive?

Reply:
If last keep alive is not updated, the PublishRequests is removed at next keepAlive